unit-tests/live/streaming/test-y16-calibration-format.py

```diff
@@ -0,0 +1,57 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:device D400*
+# This test checks streaming y16 profile
+
+import time
+import pyrealsense2 as rs
+from rspy import test, log
+from rspy.timer import Timer
+
+
+y16_streamed = False
+
+
+def close_resources(sensor):
+    """
+    Stop and Close sensor.
+    :sensor: sensor of device
+    """
+    if len(sensor.get_active_streams()) > 0:
+        sensor.stop()
+        sensor.close()
+
+
+def frame_callback(frame):
+    global y16_streamed
+    y16_streamed = True
+
+
+timer = Timer(30)
```

Comment:
Please reduce to 5.
If we don't get a frame in 5 seconds, we have another problem :)
No need to stall the LibCI on that case

Reply:
You right, we confused with this value too.

Reply:
Done, a value is 5 now.